hail/python/hail/expr/expressions/base_expression.py

```diff
@@ -754,6 +754,116 @@ def _to_relational_preserving_rows_and_cols(self):
             return x.order_by(*x.key).select(name)
         return x
 
+    def export(self, filename, delimiter='\t', header=True):
+        """Export a field to a text file
+
+        Examples
+        --------
+
+        >>> mt = small_mt
```

Comment:
put this example at the end, if at all. Common stuff first, and the global field export is not going to be common.